cs/setup/Product.wxs

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+	<Product Id="*" Name="setup" Language="1033" Version="1.0.0.0" Manufacturer="Microsoft Research"
+           UpgradeCode="ddebf366-a8ca-4192-8552-dbf1d3603ae9">
```

Comment:
We’ll put it in the release section on Github. No auto-updating. Not sure when to check for this… I’m just installing the vw.exe, the vw_json.exe and create a shortcut to a command prompt with updated PATH variable.

You think it’s needed?